ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/map/MapScreen.kt

```diff
@@ -51,18 +56,27 @@ fun MapScreen() {
         val cameraPositionState = rememberCameraPositionState {
             position = CameraPosition.fromLatLngZoom(singapore, 10f)
         }
-        GoogleMap(
-            modifier = Modifier.fillMaxSize(),
-            cameraPositionState = cameraPositionState,
-            // disabled until correct handling is implemented inside a swipe dismissable nav host
-            // possible related to https://github.com/googlemaps/android-maps-compose/issues/78
-            uiSettings = MapUiSettings(scrollGesturesEnabled = false)
-        ) {
-            Marker(
-                state = MarkerState(position = singapore),
-                title = "Singapore",
-                snippet = "Marker in Singapore"
-            )
+        Box(modifier = modifier) {
+            GoogleMap(
+                modifier = Modifier.fillMaxSize(),
+                cameraPositionState = cameraPositionState,
+            ) {
+                Marker(
+                    state = MarkerState(position = singapore),
+                    title = "Singapore",
+                    snippet = "Marker in Singapore"
+                )
+            }
+            if (onClose != null) {
+                Button(
+                    modifier = Modifier
+                        .align(Alignment.BottomCenter)
+                        .size(ButtonDefaults.SmallButtonSize),
+                    onClick = onClose,
+                ) {
+                    Text(text = "X")
```

Comment:
How about using a CompactChip here, which would take up less of the screen? Like the AM/PM chip on the TimePickerWith12HourClock demo:
`            CompactChip(
                onClick = {},
                modifier = Modifier.size(width = 50.dp, height = 24.dp),
                label = {
                    Row(
                        modifier = Modifier.fillMaxWidth(),
                        horizontalArrangement = Arrangement.Center
                    ) {
                        Text(
                            text = "X",
                            color = MaterialTheme.colors.onSecondary,
                            style = MaterialTheme.typography.button,
                        )
                    }
                },
                colors = ChipDefaults.chipColors(backgroundColor = MaterialTheme.colors.secondary),
                contentPadding = PaddingValues(vertical = 0.dp),
            )
`